yarn.lock

```diff
@@ -5671,6 +5671,14 @@
   dependencies:
     "@types/estree" "*"
 
+"@types/terser-webpack-plugin@^1.2.1":
```

Comment:
I could just use a typescript declaration file instead of including this. But would be set to `any` :(

Reply:
Probably worth not including tbh

Reply:
Other then that, do the recent changes match what you were thinking?

Reply:
@Rugvip - I've removed all deps for the `terser-webpack-plugin` so that it's pulled in as an ambient deps through webpack. I needed to add an eslint disable to get passed this though.